src/core/lib/security/credentials/google_default/google_default_credentials.cc

```diff
@@ -16,6 +16,7 @@
  *
  */
 
+#include <grpc/support/port_platform.h>
```

Comment:
Let's always include a blank line after this include, so that there's no risk of clang-format re-ordering it if new includes are added later.

Reply:
done